source/energy/conversion/grid.rst

```diff
@@ -20,7 +20,10 @@ Inter-regional energy transmission infrastructure, such as natural gas pipelines
 
 Systems Integration and Reliability
 ------------------------------------------
-The MESSAGE framework includes a single annual time period characterized by average annual load and 11 geographic regions that span the globe. Seasonal and diurnal load curves and spatial issues such as transmission constraints or renewable resource heterogeneity are treated in a stylized way in the model. The mechanism to represent power system reliability in MESSAGE which is described below, elevates the stylization of temporal resolution by introducing two concepts, peak reserve capacity and general-timescale flexibility, to the model (Sullivan et al., 2013 :cite:`sullivan_electric_2013`). To represent capacity reserves in MESSAGE, a requirement is defined that each region build sufficient firm generating capacity to maintain reliability through reasonable load and contingency events. As a proxy for complex system reliability metrics, a reserve margin-based metric was used, setting the capacity requirement at a multiple of average load, based on electric-system parameters. While many of the same issues apply to both electricity from wind and solar energy, the description below focuses on wind.
+The global MESSAGE model includes a single annual time period within each modeling year characterized by average annual load and 11 geographic regions. 
+Seasonal and diurnal load curves and spatial issues such as transmission constraints or renewable resource heterogeneity are treated in a stylized way in the model. 
+The mechanism to represent power system reliability in MESSAGE is based on (Sullivan et al., 2013 :cite:`sullivan_electric_2013`). This method elevates the stylization of temporal resolution by introducing two concepts, 
+peak reserve capacity and general-timescale flexibility (for mathematical representation see this 'Section <https://message.iiasa.ac.at/en/stable/model/MESSAGE/model_core.html#system-reliability-and-flexibility-requirements>`_). To represent capacity reserves in MESSAGE, a requirement is defined that each region build sufficient firm generating capacity to maintain reliability through reasonable load and contingency events. As a proxy for complex system reliability metrics, a reserve margin-based metric was used, setting the capacity requirement at a multiple of average load, based on electric-system parameters. While many of the same issues apply to both electricity from wind and solar energy, the description below focuses on wind.
```

Comment:
In the web page, the link appears as it is written in the code: 'Section <https://message.iiasa.ac.at/en/stable/model/MESSAGE/model_core.html#system-reliability-and-flexibility-requirements>`_).

Also in the webpage this part appears as it is written (Sullivan et al., 2013 :cite:`sullivan_electric_2013`). And the reference does not appear as a number instead cite command is there.

Reply:
Thanks @GamzeUnlu95, the typo is corrected now.